_posts/2017-05-02-textworkshop17.md

```diff
@@ -0,0 +1,24 @@
+---
+name: text-workshop-2017
+layout: post_discourse
+title: Text Analysis R Developers' Workshop 2017
+date: 2017-05-03
+authors:
+  - name: Ken Benoit
+    url: http://kenbenoit.net
+tags:
+- R
+- text analysis
+---
+
+On 21-22 April, the London School of Economics hosted the _Text Analysis Package Developers' Workshop_, a two-day event held in London that brought together developers of R packages for working with text and text-related data.  This included a wide range of applications, including string handling ([**stringi**](https://github.com/gagolews/stringi)) and tokenization (the rOpenSci-onboarded [**tokenizers**](https://github.com/ropensci/tokenizers), [**KoNLP**](https://github.com/haven-jeon/KoNLP)), corpus and text processing ([**readtext**](https://github.com/kbenoit/readtext), [**tm**](http://tm.r-forge.r-project.org), [**quanteda**](http://quanteda.io), and [**qdap**](https://github.com/trinker/qdap)), natural language processing (NLP) such as part of speech and dependency tagging ([**cleanNLP**](https://github.com/statsmaths/cleanNLP), [**spacyr**](https://github.com/kbenoit/spacyr)), and the statistical analysis of textual data ([**stm**](http://www.structuraltopicmodel.com), [**text2vec**](https://github.com/dselivanov/text2vec), and [**koRpus**](https://reaktanz.de/?c=hacking&s=koRpus)) -- although this list is hardly complete.  The main objective was to bring together experts working on various aspects of text processing and text analysis using R, to discuss common challenges and identify collaborative solutions.  
+
+The workshop was semi-structured, somewhere in between a traditional agenda-driven workshop and an "[unconference](http://unconf17.ropensci.org)".  On Day 1 and in the first session of Day 2, we held a set of seven topically focused roundtables, with between 3-4 participants plus an appointed moderator.  Organized around themes or problems, such as segmenting and tokenizing text, or trying to parse linguistic structure and parts of speech, each session lasted about an hour and features 3-4 participants and a moderator. (The full list is [here](https://github.com/ropensci/textworkshop17/blob/master/preliminary_schedule.md#schedule).) This gave everyone a chance to participate in a structured event, and served to guide discussion.  This proved invaluable in identifying challenges, especially the final roundtable on Saturday morning to discuss package interoperability, which had been aided greatly by the overall discussion that took place before across the first six roundtables.  (A group dinner at a [local brewpub](http://www.templebrewhouse.com/brewery/) the evening before also helped spur this discussion.)
```

Comment:
omit "...and features 3-4 participants and a moderator" right before "(The full list is [here]..." as it's a repeat of above